### PR TITLE
v1.19 Windows OS version support + containerd doc updates

### PR DESCRIPTION
**Pull request by marosset**
Doc PR for https://github.com/kubernetes/enhancements/issues/1001 for 1.19 updates

### CODE REVIEW

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Docker EE-basic 18.09+ is the recommended container runtime for Windows Server 2
 
 ##### CRI-ContainerD
 
-{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
 
 ContainerD is an OCI-compliant runtime that works with Kubernetes on Linux. Kubernetes v1.18 adds support for {{< glossary_tooltip term_id="containerd" text="ContainerD" >}} on Windows. Progress for ContainerD on Windows can be tracked at [enhancements#1001](https://github.com/kubernetes/enhancements/issues/1001).
```
Comment (sftim) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:113]:
```suggestion
[containerd](https://containerd.io/) is an OCI-compliant runtime that works with Kubernetes. You can
use containerd with Linux nodes and, as a beta feature, on Windows.
```

**Comment by divya-mohan0209** [comment]
@marosset : Please could you confirm in light of the revised deadlines if this is right now ready for review?

**Comment by marosset** [comment]
/assign @michmike 
/cc @jsturtevant

(Sorry for the last minute PR!)

**Comment by divya-mohan0209** [comment]
Hello @marosset @michmike : Have we managed to get this reviewed & signed off from a doc and tech perspective? We need to ensure that we have both of them to successfully merge it.

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 2]
```diff
@@ -104,22 +113,15 @@ Pods, Controllers and Services are critical elements to managing Windows workloa
...
-ContainerD is an OCI-compliant runtime that works with Kubernetes on Linux. Kubernetes v1.18 adds support for {{< glossary_tooltip term_id="containerd" text="ContainerD" >}} on Windows. Progress for ContainerD on Windows can be tracked at [enhancements#1001](https://github.com/kubernetes/enhancements/issues/1001).
-
-{{< caution >}}
-
-ContainerD on Windows in Kubernetes v1.18 has the following known shortcomings:
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
 
-* ContainerD does not have an official release with support for Windows; all development in Kubernetes has been performed against active ContainerD development branches. Production deployments should always use official releases that have been fully tested and are supported with security fixes.
-* Group-Managed Service Accounts are not implemented when using ContainerD - see [containerd/cri#1276](https://github.com/containerd/cri/issues/1276).
+{{< glossary_tooltip term_id="containerd" text="ContainerD" >}} 1.4.0-beta.1+ can also be used as the container runtime for Windows Kubernetes nodes.
```
Comment (daschott) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:122]: Perhaps add a hyperlink to https://github.com/containerd/containerd/releases/tag/v1.4.0-beta.1 ?

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 3]
```diff
@@ -54,6 +59,10 @@ The Windows Server Host Operating System is subject to the [Windows Server ](htt
 Windows containers with process isolation have strict compatibility rules, [where the host OS version must match the container base image OS version](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility). Once we support Windows containers with Hyper-V isolation in Kubernetes, the limitation and compatibility rules will change.
 {{< /note >}}
 
+#### Compute
+
+From an API and kubectl perspective, Windows containers behave in much the same way as Linux-based containers. However, there are some notable differences in key functionality which are outlined in the limitation section.
```
Comment (sftim) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:64]: nit: hyperlink to the list of limitations.

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 4]
```diff
@@ -33,17 +33,22 @@ In this document, when we talk about Windows containers we mean Windows containe
...
+| *Kubernetes v1.14* | Windows Server 2019 | Windows Server ver 1809 |
+| *Kubernetes v1.15* | Windows Server 2019 | Windows Server ver 1809 |
+| *Kubernetes v1.16* | Windows Server 2019 | Windows Server ver 1809 |
+| *Kubernetes v1.17* | Windows Server 2019 | Windows Server ver 1809 |
+| *Kubernetes v1.18* | Windows Server 2019 | Windows Server ver 1809, Windows Server ver 1903, Windows Server ver 1909 |
+| *Kubernetes v1.19* | Windows Server 2019 | Windows Server ver 1909, Windows Server ver 2004 |
 
+{{< note >}}
+Information on the different Windows Server servicing channels including their support models can be found at [Windows Server servicing channels](https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19).
+{{ /note >}}
```
Comment (sftim) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:51]: This shortcode is invalid. I think you mean:
```suggestion
{{< /note >}}
```

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md (hunk 2 above)
Comment (jsturtevant) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md]: I updated to point to a doc that shows how to configure on a windows node.  does that covers this sufficiently?

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md (hunk 3 above)
Comment (jsturtevant) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md]: thanks! resolved

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md (hunk 4 above)
Comment (jsturtevant) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md]: thanks! resolved

**Comment by jsturtevant** [comment]
Ready for technical review 

/cc @michmike @craiglpeters @immuzz  @ddebroy @daschott

**Comment by celestehorgan** [comment]
from a content/grammar/etc. perspective 

/lgtm

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 5]
```diff
@@ -33,17 +33,22 @@ In this document, when we talk about Windows containers we mean Windows containe
 
 ### Supported Functionality
 
-#### Compute
-
-From an API and kubectl perspective, Windows containers behave in much the same way as Linux-based containers. However, there are some notable differences in key functionality which are outlined in the limitation section.
+#### Windows OS Version Support
 
-Let's start with the operating system version. Refer to the following table for Windows operating system support in Kubernetes. A single heterogeneous Kubernetes cluster can have both Windows and Linux worker nodes. Windows containers have to be scheduled on Windows nodes and Linux containers on Linux nodes.
+ Refer to the following table for Windows operating system support in Kubernetes. A single heterogeneous Kubernetes cluster can have both Windows and Linux worker nodes. Windows containers have to be scheduled on Windows nodes and Linux containers on Linux nodes.
```
Comment (michmike) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:38]: extra space before the word "refer"

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 6]
```diff
@@ -399,6 +399,27 @@ containerd config default > /etc/containerd/config.toml
 systemctl restart containerd
 ```
 {{% /tab %}}
+{{% tab name="Windows (Powershell)" %}}
```
Comment (michmike) [inline content/en/docs/setup/production-environment/container-runtimes.md:402]: nit. PowerShell should have "s" capitalized

**Comment by annajung** [comment]
Hi @marosset, 👋 1.19 doc shadow here. A friendly reminder that docs deadline is tomorrow. Pinging you to address the review suggestions above so we can get approvals from both tech/docs to merge this in! thank you

**Comment by jsturtevant** [comment]
I pushed some updates but it appears PR's are not syncing with with the branch due to github issue.  Will check back in a few after this incident is resolved

**Comment by jsturtevant** [comment]
@annajung  I will be finishing up the PR so you can ping me if there are any updates

**Pull request merged**